modules/nf-core/modkit/pileup/tests/main.nf.test

```diff
@@ -0,0 +1,378 @@
+nextflow_process {
+
+    name "Test Process MODKIT_PILEUP"
+    script "../main.nf"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "modkit"
+    tag "modkit/pileup"
+    process "MODKIT_PILEUP"
+
+    test("[bam, bai], [], []") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id: 'bc_anchored' ],
+                    file('/media/ssd_4tb/felix/projects/fellen31/test-datasets/data/delete_me/modkit/bc_anchored_10_reads.haplotyped.sorted.bam', checkIfExists: true),
+                    file('/media/ssd_4tb/felix/projects/fellen31/test-datasets/data/delete_me/modkit/bc_anchored_10_reads.haplotyped.sorted.bam.bai', checkIfExists: true)
```

Comment:
The tests are very comprehensive, but you cannot just link data from your personal portable drive, as nobody (especially not AWS runners) will run the tests with your hardware. Consult #5281 for an example of how to use test data in module tests.

Reply:
Hi, thanks for the review. I'm waiting for https://github.com/nf-core/test-datasets/pull/1224, that's why I kept the PR as a draft. 

Will update after the testdata PR is reviewed and merged.